src/components/verify/document-renderer.tsx

```diff
@@ -65,14 +66,32 @@ export const DocumentRenderer: React.FunctionComponent<DocumentRendererProps> =
           templates={templates}
           selectedTemplate={selectedTemplate}
           onSelectTemplate={(selectedTemplate) => setSelectedTemplate(selectedTemplate)}
+          onPrint={() => {
+            toFrame?.(print());
+          }}
         />
       )}
       <FrameConnector
+        css={css`
+          @media print {
+            display: none;
+          }
+        `}
         style={{ height: `${height}px`, width: "100%", border: "0px" }}
         source={`${typeof document.$template === "object" ? document.$template.url : document.$template}`}
         onConnected={onConnected}
         dispatch={fromFrame}
       />
+      <div
+        css={css`
+          @media print {
+            display: block;
+          }
+        `}
+        className="hidden bg-red-100 border-t-4 border-red-500 text-red-700 p-4 w-full text-center break-all mt-8"
+      >
```

Comment:
think can remove hidden

Reply:
nah otherwise the message will be visible on the website. The goel is to display it only when user uses the browser print button

Reply:
right